.circleci/config.yml

```
@@ -28,13 +34,9 @@ jobs:
           name: adding-golangci-lint
           command: wget -O - -q https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.21.0
       - checkout
-      - *restore_deps
-      - run: go get -u github.com/golang/dep/cmd/dep
-      - run: dep ensure
-      - save_cache:
-          key: gopkg-{{ .Branch }}-{{ checksum "Gopkg.lock" }}
-          paths:
-            - /go/src/github.com/honeydipper/honeydipper/vendor
+      - *restore_mod
+      - run: go build -v ./...
```

```suggestion
      - run: go install -v ./...
```

I would recommend doing a rebase so changes can be squashed into a single commit that meets the semantic version requirements.

---

Here I'm doing `go build` to replicate the previous `dep ensure`, I've changed the `integration` step below to use `go install`

---

I might be wrong, but I think `go get -u -t ./...` is equivalent to `dep ensure` in go modules term.

---

That will update dependencies, I'm assuming we want a hermetic build in CI and humans will update dependencies and submit PRs? Per https://github.com/golang/go/wiki/Modules#how-to-use-modules `go build will automatically add missing or unconverted dependencies as needed to satisfy imports for this particular build invocation`  (same for go install as it performs a build)